packages/engine-test-utils/src/__tests__/dendron-cli/commands/seedCLICommand.spec.ts

```diff
@@ -205,7 +205,7 @@ describe("add", () => {
             fpath: path.join(wsRoot, "dendron.code-workspace"),
             snapshot: true,
           },
-          `${id}/vault`
+          path.join(`${id}`, "vault").replace("\\", "\\\\")
```

Comment:
might be good to wrap in a utility function. `eg. engine-test-utils/src/utils/workspace.ts`